components/gsma-authenticators/src/main/java/com/wso2telco/gsma/authenticators/voice/VoiceCallAuthenticator.java

```diff
@@ -0,0 +1,265 @@
+
+    private final String isUserEnrolledUrl = "https://poc.vsservic.es/pwr/isUserActive";
+    private final String verifyUserUrl = "https://poc.vsservic.es/pwr/authenticateUser";
+    private final String onBoardUserUrl = "https://poc.vsservic.es/pwr/onboardUser";
+
+
+    @Override
+    public AuthenticatorFlowStatus process(HttpServletRequest request, HttpServletResponse response, AuthenticationContext context) throws AuthenticationFailedException, LogoutFailedException {
+
+        log.info("Yepeeeeeeeeeeeeeeee");
+        //return super.process(request, response, context);
+
+        super.process(request, response, context);
+        if (context.isLogoutRequest()) {
+            return AuthenticatorFlowStatus.SUCCESS_COMPLETED;
+        }
+        else {
+            boolean isFlowCompleted = (boolean) context.getProperty(IS_FLOW_COMPLETED);
+
+            if (isFlowCompleted) {
+                return AuthenticatorFlowStatus.SUCCESS_COMPLETED;
+            } else {
+                return AuthenticatorFlowStatus.INCOMPLETE;
+            }
+        }
+
+    }
+
+
+    @Override
+    protected void initiateAuthenticationRequest(HttpServletRequest request, HttpServletResponse response, AuthenticationContext context) throws AuthenticationFailedException {
+        super.initiateAuthenticationRequest(request, response, context);
+        log.info("#### #### Initiating authentication request from Voice Call Authentication");
+
+        String queryParams = FrameworkUtils.getQueryStringWithFrameworkContextId(context.getQueryParams(),
+                context.getCallerSessionKey(), context.getContextIdentifier());
+        ApplicationConfig applicationConfig = context.getSequenceConfig().getApplicationConfig();
+        Map<String, String> paramMap = Util.createQueryParamMap(queryParams);
+        String msisdn = (String) context.getProperty(MSISDN);
+        String clientId = paramMap.get(CLIENT_ID);
+        String applicationName = applicationConfig.getApplicationName();
+        String sessionKey = context.getCallerSessionKey();
+
+        log.info("MSISDN : " + msisdn);
+        log.info("Client ID : " + clientId);
+        log.info("Application name : " + applicationName);
+        log.info("sessionKey  : " + sessionKey);
+
+        ValidSoftJsonHelper validSoftJsonHelper = new ValidSoftJsonHelper();
+        validSoftJsonHelper.setIsUserActiveRequestJson(sessionKey, msisdn);
+        validSoftJsonHelper.setUserRegistrationAndAuthenticationJson(sessionKey, msisdn);
+        StringEntity postData = null;
+        StringEntity verifyAndRegistartionPostData = null;
+        try {
+            postData = new StringEntity(validSoftJsonHelper.getIsUserActiveRequestJson());
+            verifyAndRegistartionPostData = new StringEntity(validSoftJsonHelper.getUserRegistrationAndAuthenticationJson());
+            log.info("Json Object : " + postData);
+        } catch (UnsupportedEncodingException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1148)